src/company_type.h

```diff
@@ -49,7 +49,11 @@ class CompanyMask : public BaseBitSet<CompanyMask, CompanyID, uint16_t> {
 };
 
 struct Company;
-typedef uint32_t CompanyManagerFace; ///< Company manager face bits, info see in company_manager_face.h
+
+struct CompanyManagerFace {
+	uint style = 0; ///< Company manager face style.
+	uint32_t bits = 0; ///< Company manager face bits, info see in company_manager_face.h
```

Comment:
Typo or maybe I misunderstood. 🙂 

```suggestion
	uint32_t bits = 0; ///< Company manager face bits, see company_manager_face.h
```

Reply:
Just an old comment written by a non-native speaker.